<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>702</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="basicGroupBox">
    <property name="geometry">
     <rect>
      <x>14</x>
      <y>12</y>
      <width>411</width>
      <height>350</height>
     </rect>
    </property>
    <property name="title">
     <string>Font Information</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="formLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>390</width>
       <height>316</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::FieldsStayAtSizeHint</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="fontLabel">
        <property name="minimumSize">
         <size>
          <width>115</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Font</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="fontbutton">
        <property name="text">
         <string>Pick Font</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="characterSetLabel">
        <property name="minimumSize">
         <size>
          <width>115</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Character Set:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPlainTextEdit" name="characterSetBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>192</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="lineWrapMode">
         <enum>QPlainTextEdit::WidgetWidth</enum>
        </property>
        <property name="plainText">
         <string>abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ1234567890[]'#;,./\`¬!&quot;£$%^&amp;*()_+{}:@~?&gt;&lt;|</string>
        </property>
        <property name="backgroundVisible">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="fillColorLabel">
        <property name="minimumSize">
         <size>
          <width>115</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Fill Color</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QFrame" name="fillColor">
        <property name="minimumSize">
         <size>
          <width>230</width>
          <height>32</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QPushButton" name="fillColorButton">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>1</y>
           <width>81</width>
           <height>32</height>
          </rect>
         </property>
         <property name="text">
          <string>Select</string>
         </property>
        </widget>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="fillAngleLabel">
        <property name="minimumSize">
         <size>
          <width>115</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Fill Angle</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QDoubleSpinBox" name="fillAngleBox"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="strokeGroupBox">
    <property name="geometry">
     <rect>
      <x>14</x>
      <y>560</y>
      <width>411</width>
      <height>110</height>
     </rect>
    </property>
    <property name="title">
     <string>Stroke</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="formLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>364</width>
       <height>74</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="strokeColorLabel">
        <property name="minimumSize">
         <size>
          <width>115</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Stroke Color</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QFrame" name="strokeColor">
        <property name="minimumSize">
         <size>
          <width>230</width>
          <height>32</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QPushButton" name="strokeColorButton">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>1</y>
           <width>82</width>
           <height>32</height>
          </rect>
         </property>
         <property name="text">
          <string>Select</string>
         </property>
        </widget>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="strokeWidthLabel">
        <property name="minimumSize">
         <size>
          <width>115</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Stroke Width</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="strokeWidthBox"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="dropShadowGroupBox">
    <property name="geometry">
     <rect>
      <x>14</x>
      <y>370</y>
      <width>411</width>
      <height>180</height>
     </rect>
    </property>
    <property name="title">
     <string>Drop Shadow</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="formLayoutWidget_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>360</width>
       <height>142</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout_4">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::FieldsStayAtSizeHint</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="shadowColorLabel">
        <property name="minimumSize">
         <size>
          <width>115</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Shadow Color</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QFrame" name="shadowColor">
        <property name="minimumSize">
         <size>
          <width>230</width>
          <height>32</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QPushButton" name="shadowColorButton">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>1</y>
           <width>81</width>
           <height>32</height>
          </rect>
         </property>
         <property name="text">
          <string>Select</string>
         </property>
        </widget>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="shadowAngleLabel">
        <property name="minimumSize">
         <size>
          <width>115</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Shadow Angle</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="shadowAngleBox">
        <property name="minimum">
         <double>-360.000000000000000</double>
        </property>
        <property name="maximum">
         <double>360.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="shadowSizeLabel">
        <property name="minimumSize">
         <size>
          <width>115</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Shadow Size</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDoubleSpinBox" name="shadowSizeBox"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="shadowDistanceLabel">
        <property name="minimumSize">
         <size>
          <width>115</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Shadow Distance</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QDoubleSpinBox" name="shadowDistanceBox"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QPaintArea" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>30</y>
      <width>351</width>
      <height>501</height>
     </rect>
    </property>
    <zorder>verticalLayoutWidget</zorder>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>550</y>
      <width>160</width>
      <height>134</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="exportSeperateButton">
       <property name="text">
        <string>Export Individual</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="exportGroupButton">
       <property name="text">
        <string>Export Group</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="saveSettingsButton">
       <property name="text">
        <string>Save Settings</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="loadSettingsButton">
       <property name="text">
        <string>Load Settings</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QPaintArea</class>
   <extends>QWidget</extends>
   <header>qpaintarea.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
